tests/test_area.py

```diff
@@ -61,16 +61,6 @@ def setUp(self):
     def tearDown(self):
         GlobalConfig.market_count = 1
 
-    def test_respective_area_grid_fee_is_applied(self):
```

Comment:
I guess this test has to be reimplemented in the test_market, right?

Reply:
https://github.com/gridsingularity/d3a/pull/443/files#diff-f46f8a6e9b1f4757a0e767f3306499d2R154
I guess, its already taken care of in tests/test_inter_area_agent.py